### PR TITLE
F9

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
teach new devs for good

debug break via f9

### CODE REVIEW

File: docs/core/tutorials/with-visual-studio-code.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
     ![Selecting .NET Core](media/with-visual-studio-code/selectcore.png)
 
-5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9.
+5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9, or put mouse cursor on line 9 and press <kbd>F9</kbd>.
```
Comment (svick) [inline docs/core/tutorials/with-visual-studio-code.md]: Where the breakpoint is placed when you press <kbd>F9</kbd> is not controlled by the mouse cursor, but by the text cursor. So I think it's confusing to talk about the mouse cursor here.

File: docs/core/tutorials/with-visual-studio-code.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
     ![Selecting .NET Core](media/with-visual-studio-code/selectcore.png)
 
-5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9.
+5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9, or move text cursor into line 9 in the editor and then press <kbd>F9</kbd>.
```
Comment (rpetrusha) [inline docs/core/tutorials/with-visual-studio-code.md]: move text cursor into --> move the text cursor onto
and then press --> and press

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for contributing to the dotnet/docs repo, @dzmitry-lahoda, and for adding an alternate way to set a breakpoint. Ordinarily, our introductory tutorials try to show just one way to perform a UI interface action. Since customers sometimes have difficulty setting a breakpoint by clicking in the editor margin, an exception is warranted in this case. I would, however, like to you make two additional corrections before we merge your PR.

**Review by rpetrusha — APPROVED** [review]
Thank you, @dzmitry-lahoda, for making the additional changes. We'll merge your PR now. Your changes should appear live on docs.microsoft.com in the next day or two.

**Pull request merged**